core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -242,4 +242,30 @@
    * @since 1.8.0
    */
   long getBatchTimeout(TimeUnit timeUnit);
+
+  /**
+   * Sets the name of the classloader context on this scanner
```

Comment:
Would be nice to add some text here pointing users to the Admin manual for more info.   Don't of a way to link with a URL, but we could still say `go look at section X in the admin manual for more info`.